source/LMPlant.h

```diff
@@ -0,0 +1,124 @@
+//
+//  LMPlant.h
+//  Lumia
+//
+//  Created by Ben Dacek on 3/9/21.
+//  Copyright © 2021 Cornell Game Design Initiative. All rights reserved.
+//
+
+#ifndef LMPlant_h
+#define LMPlant_h
+#include <cugl/cugl.h>
+#include <cugl/physics2/CUBoxObstacle.h>
+#include <cugl/scene2/graph/CUWireNode.h>
+
+#pragma mark -
+#pragma mark Plant Model
+class Plant : public cugl::physics2::WheelObstacle {
```

Comment:
the plant should not be a wheel obstacle... it should just be a fixture, so it does not stop the player, but still contacts with it. also, the plant designer gave us looks more like a polygon?